packages/mobile/src/app/App.tsx

```diff
@@ -18,16 +19,14 @@ import Navigator from 'src/navigator/NavigatorWrapper'
 import { persistor, store } from 'src/redux/store'
 import Logger from 'src/utils/Logger'
 
+useScreens()
```

Comment:
This turns on RN screens - we had some issues with copying and pasting before, but seems fine to me now

Reply:
https://twitter.com/janicduplessis/status/1039979591815897088?s=21